test/player_src_equals_integration.js

```diff
@@ -351,4 +357,22 @@ describe('Player Src Equals', () => {
     // point ensures it is safe to interact with the media element.
     await ready;
   }
+
+  /**
+   * @param {string} contentUri
+   * @param {number} startTime
+   * @return {!Promise}
+   */
+  async function loadWithSrcEqualsAndStartTime(contentUri, startTime) {
```

Comment:
You could also use `loadWithSrcEquals` and add an optional argument that defaults to `null` or `undefined`.

Reply:
Changed!